earth_enterprise/src/fusion/autoingest/tools/geaddtoimageryproject.cpp

```diff
@@ -35,20 +35,40 @@ usage(const std::string &progn, const char *msg = 0, ...)
     fprintf(stderr, "\n");
   }
 
-  fprintf(stderr,
-          "\nusage: %s [options] -o <projectname> {[--maxlevel <level>] "
-          "<insetresource>}...\n"
-          "   Supported options are:\n"
-          "      --help | -?:      Display this usage message\n"
-          "      --mercator : use mercator projection\n"
-          "      --flat : use flat(Plate Carre) projection (default)\n"
-          "      --historical_imagery : change this project to be a historical\n"
-          "                             imagery project\n"
-          "      --no_historical_imagery : make this a normal project\n"
-          "                         (i.e., not a historical imagery project)\n"
-          "   By default, new projects are NOT time machine projects"
-          " unless --historical_imagery is specified.\n",
-          progn.c_str());
+  if (AssetDefs::Terrain == AssetType){
```

Comment:
I created a new terrain project using the following command `genewterrainproject -o terrainprojecttest --terrain_overlay`: no error or warning were posted about missing start level and min level resource parameters and values. When i opened that project in Fusion "Build Overlay" checkbox was not enabled. If the command did not error or post a warning, i was excepting that overlay will be used with default parameters. If not we should probably post an error or warning.

Reply:
@tst-nfarah I'm not sure what's happening there. I copied that command straight from your comment and ran it on my development machine, and when I look at the project in Fusion, the 'Build "Overlay"' checkbox is enabled, and "Start Level" and "Resources Min Level" have the default values. I'll do a pull and rebuild, then try it again to see if I see the same thing.

Reply:
After installing the new built branch, i did not have this issue any more. I had previous Fusion installed and it appears that using the commands built with this branch is not enough to get to a clean state. This is now resolved.